images/Images/ImageFITS2Converter.cc

```diff
@@ -804,13 +808,13 @@ Bool ImageFITSConverter::ImageToFITSOut(
 	// and make Stokes the very last if requested
 	//
 	IPosition shape = image.shape();
-	IPosition newShape = shape;
+        fhi.newShape = shape;
```

Comment:
Should have been indented with tabs here, like the rest of the file. Sorry for nitpicking..